Sem. 09/Pract.10/Task17.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+using namespace std;
+
+constexpr int MAX_LENGTH_STR = 10000;
+constexpr int MAX_SIZE_MATR = 100;
+constexpr int MAX_SYMBOLS_IN_WORD = 1024;
+
+void stringToMatrix(const char* str, char matrix[MAX_SIZE_MATR][MAX_SYMBOLS_IN_WORD], int& row) {
+    if (!str)
+        return;
+    int ind = 0;
+
+    while (*str) {
+        if (*str == ' ' && *(str - 1) != ' ') {
+            row++;
+            ind = 0;
+        }
+        else {
+            matrix[row][ind++] = *str;

```

Comment:
You must have '\0' at the end of each word. 
Also, it does not work when several spaces are separating the word.
Check this [solution](https://github.com/nikola354/Introduction-to-programming-FMI/blob/main/Sem.%2009/Pract.10/Task17.cpp).